docs/source/install_steps.rst

```diff
@@ -14,3 +18,10 @@ The Quipucords server is delivered as a container image that runs on your server
 .. include:: install_server_fedora.rst
 .. include:: install_container_image.rst
 .. include:: configure.rst
+
+Configuring Quipucords
```

Comment:
I think that we should remove this section entirely. If you view it in readthedocs you can see that the section ```Changing the Default Password for ...``` is literally above it, as is a link to ```Logging in to the Quipucords Server```.

Reply:
I agree with Ashley